core/app/modules/import-export/assets/js/ui/title/title.js

```diff
@@ -0,0 +1,49 @@
+import Utils from 'elementor-app/utils/utils.js';
+
+import Heading from 'elementor-app/ui/atoms/heading';
+import Text from 'elementor-app/ui/atoms/text';
+
+import './title.scss';
+
+export default function Title( props ) {
```

Comment:
1. Isn't it a general component?
2. Mind the prefix throughout this file.

Reply:
1. As agreed, for now, we'll keep it in the import-export level.
2.